lib/views/widgets/navigation/bottom_navbar.dart

```diff
@@ -62,7 +62,7 @@ class BottomNavBar extends HookConsumerWidget {
                 NavigationButton(
                   icon: Icons.notifications_none_outlined,
                   activeIcon: Icons.notifications_active,
-                  route: Routes.notifications,
+                  route: null,
```

Comment:
@ErickDoroy 
Can you just comment this out. As reference in the future

Reply:
hai ok miss